### DISCUSSION TITLE
Only using 11-12 threads on 128 thread system

### DISCUSSION THREAD

**Discussion by pletharoe** [madMAx43v3r/chia-plotter#444] [Q&A]
3995wx
512GB RAM
Clear linux

./chia_plot -n200 -r 128 -t /media/dom/ssd1/ -2 /media/ram/ -d /media/ssd2/ -p [BLAHBLAH] -f [BLAHBLAH]

Why is it only using about 11-12 threads?

**Comment by ArianHiwa** [comment]
why you dont run it 10 X 12 threads?

**Comment by madMAx43v3r** [comment]
`-n200` => `-n 200` (notice the space after `n`)
